src/cli/gui.js

```diff
@@ -40,7 +40,6 @@ export default class GUI {
      */
     finish() {
         if (this.ci) {
-            // eslint-disable-next-line no-console
```

Comment:
It's not for `svglint` only. Sindre and I were focusing on CLI apps before. Imagine the rule spamming on our CLI sources.

Reply:
Ok, but can be added to this project, right?

Reply:
> In that case, would you mind to add it?

Both are OK with me. We can add it since the code only uses one console.log. But the rule should be removed if we want to use more console.log in the future.

Reply:
I think that the no-console rule should be keep for now.

Reply:
np. Do we need this rule for our main repo? I can add it to there as well.

Reply:
Yes, please open a PR for that 🙏🏼